projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/proxy/CommunityApiController.kt

```diff
@@ -96,6 +112,13 @@ class CommunityApiController(
         @RequestParam(required = false, defaultValue = "true") excludeSoftDeleted: Boolean
     ): Any {
 
+        sendComparisonReport(
+            mapOf("crn" to crn) +
+                mapOf("convictionId" to convictionId) +
+                mapOf("activeOnly" to activeOnly) +
+                mapOf("excludeSoftDeleted" to excludeSoftDeleted), Uri.CONVICTION_REQUIREMENTS, request
```

Comment:
You should be able combine these mapOf calls here:
```suggestion
            mapOf("crn" to crn, "convictionId" to convictionId, "activeOnly" to activeOnly, "excludeSoftDeleted" to excludeSoftDeleted), Uri.CONVICTION_REQUIREMENTS, request
```